frontend/src/metabase/query_builder/components/NativeQueryEditor/CodeMirrorEditor/util.tsx

```diff
@@ -225,9 +227,12 @@ export function matchCardIdAtCursor(
   return parsedId;
 }
 
-export function referencedQuestionIds(query: Lib.Query): CardId[] {
-  return Object.values(Lib.templateTags(query) ?? {})
```

Comment:
I verified that `Lib.templateTags` cannot return `null`:

<img width="1027" alt="Screenshot 2025-02-05 at 10 45 18" src="https://github.com/user-attachments/assets/ba063a6a-3066-467c-92c1-33004f22be07" />

The signature already doesn't include `null` btw.